src/main/java/dev/jbang/DependencyUtil.java

```diff
@@ -177,7 +219,7 @@ public String decodeEnv(String value) {
 	}
 
 	public static boolean looksLikeAGav(String candidate) {
```

Comment:
maybe change the name to looksLikeAFullGav ?

Don't you need to look for gav (not full) anymore?

Reply:
this method is to recognize GAV's on command line so I think its named appropriately. better if we remove the need to have a full GAV pattern.